TransactionMobile/TransactionMobile/App.xaml.cs

```diff
@@ -45,6 +47,11 @@ public partial class App : Application
         /// </summary>
         public static Guid MerchantId;
 
+        /// <summary>
+        /// The contract products
+        /// </summary>
+        public static List<ContractProductModel> ContractProducts;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/MobileApplication/pullRequest?prid=5918284)